sql/core/src/test/resources/sql-tests/results/udf/postgreSQL/udf-select_having.sql.out

```diff
@@ -143,7 +143,7 @@ SELECT udf(a) FROM test_having HAVING udf(min(a)) < udf(max(a))
 struct<>
 -- !query output
 org.apache.spark.sql.AnalysisException
-grouping expressions sequence is empty, and 'spark_catalog.default.test_having.`a`' is not an aggregate function. Wrap '(min(spark_catalog.default.test_having.`a`) AS `min(a#x)`, max(spark_catalog.default.test_having.`a`) AS `max(a#x)`)' in windowing function(s) or wrap 'spark_catalog.default.test_having.`a`' in first() (or first_value) if you don't care which value you get.;
+grouping expressions sequence is empty, and 'spark_catalog.default.test_having.`a`' is not an aggregate function. Wrap '(min(spark_catalog.default.test_having.`a`) AS `min(a)`, max(spark_catalog.default.test_having.`a`) AS `max(a)`)' in windowing function(s) or wrap 'spark_catalog.default.test_having.`a`' in first() (or first_value) if you don't care which value you get.;
```

Comment:
We need to hide ids even in error messages, too?

Reply:
I think the `flatArgument/flatArgumentsString` is tightly bind with `toString` of `Expression/AggregateExpression`, which will commonly affect all of them. I'll try more to eliminate the impact, thanks!

Reply:
The error message explicitly called `map(_.sql)` instead of default `toString`. The `Alias.sql` is using `name` field which has already been formatted by `flatArguments` when constructing `Alias`. So this is also following the `flatArgument` framework.
As the error message is intended to suggest user with a sql snippet, maybe it's better to not includ `#exprId` anyway? Thanks.